lib/helper/Playwright.js

```diff
@@ -2655,6 +2661,265 @@ class Playwright extends Helper {
   async stopMockingRoute(url, handler) {
     return this.browserContext.unroute(...arguments);
   }
+
+  /**
+   * Starts recording of network traffic.
+   * This also resets recorded network requests.
+   *
+   * @return {Promise<void>}
+   */
+  async startRecordingTraffic() {
+    this._flushTraffic();
+    this.recording = true;
+    this.recordedAtLeastOnce = true;
+
+    this.page.on('requestfinished', async (request) => {
+      const information = {
+        url: request.url(),
+        method: request.method(),
+        requestHeaders: request.headers(),
+        requestPostData: request.postData(),
+      };
+
+      this.debugSection(`REQUEST: ${JSON.stringify(information)}`);
+
+      information.requestPostData = JSON.parse(information.requestPostData);
+      this.requests.push(information);
+      return this._waitForAction();
+    });
+  }
+
+  /**
+  *  Grab the recording network traffics
+  *
+  * @return { Promise<Array<any>> }
+  *
+  */
+  async grabRecordingTraffic() {
```

Comment:
I would rather call it `I.grabRecordedTraffic`. 

The reality is that this method just returns the traffic requests recorded so far. Nothing more, nothing less.
When I see name `I.grabRecordingTraffic` I am bit unsure if receive some kind of stream object, because the active meaning of the word "recording".